src/components/Grid/Grid.tsx

```diff
@@ -46,6 +52,23 @@ const Grid: FC<Props> = ({
     getWidgetsPosition(state, widgetsId)
   );
 
+  const { setGridSize, droppableWidget, editorPubSub } = useContext(
+    EditorContext
+  );
+  const [cover, showCover] = useState(null);
+  const [isResize, setResize] = useState(false);
+
+  const onResizeStart = () => {
+    setResize(true);
+  };
+
+  const onResizeStop = (_items, item) => {
+    const { i: id } = item;
+    setResize(false);
+    editorPubSub.publish(RESIZE_WIDGET_EVENT, { id });
+    if (onWidgetResize) onWidgetResize;
```

Comment:
_onWidgetResize_ handler should be invoked in this line to update widget positions in grid.